bower.json

```diff
@@ -14,5 +14,20 @@
     "angular-ui-router": "~0.2.18",
     "bootstrap": "~3.3.6",
     "owasp-password-strength-test": "~1.3.0"
+  },
+  "overrides": {
+    "bootstrap": {
+      "main": [
+        "dist/css/bootstrap.css",
+        "dist/css/bootstrap-theme.css",
+        "less/bootstrap.less"
+      ]
+    },
+    "jquery": {
+      "main": []
+    }
+  },
+  "resolutions": {
+    "angular": "~1.3"
```

Comment:
Interesting, I'm not able to reproduce the issue on my side. Even after doing `bower cache clean`. I guess we can remove the resolutions config here, and roll with it.

@itelo Is there any particular reason/need for it to be here?